python/tchannel/tornado/tchannel.py

```diff
@@ -46,28 +44,22 @@ def get_peer(self, hostport):
 
         raise tornado.gen.Return(peer)
 
-    @tornado.gen.coroutine
-    def make_in_connection(self, port):
-        self.inbound_server.listen(port)
-
     def request(self, hostport):
-        return TChannelClientOperation(hostport, self)
+        return TChannelClientOperation(hostport, self,)
```

Comment:
Unnecessary trailing comma. Generally they're added if the call is spread across multiple lines (with one parameter per line)